raiden-dapp/src/components/account/Withdrawal.vue

```diff
@@ -135,18 +138,28 @@ import Spinner from '@/components/icons/Spinner.vue';
   },
   computed: {
     ...mapState(['tokens', 'raidenAccountBalance']),
+    ...mapGetters(['udcToken']),
   },
 })
 export default class Withdrawal extends Mixins(BlockieMixin) {
   tokens!: Tokens;
   raidenAccountBalance!: string;
+  udcToken!: Token;
   balances: Token[] = [];
   loading: boolean = true;
   withdrawing: boolean = false;
   withdraw: Token | null = null;
 
   async mounted() {
     this.balances = await this.$raiden.getRaidenAccountBalances();
+
+    if ((this.udcToken.balance as BigNumber).gt(Zero)) {
```

Comment:
you're not testing the raiden account balance for udcToken, you're testing just udcToken, which probably is the main account balance

Reply:
Yes, one sec, I'm making a new PR. Stuff accidentally committed the wrong stuff. :)